securedrop_client/storage.py

```diff
@@ -21,7 +21,7 @@
 """
 import logging
 from dateutil.parser import parse
-from securedrop_client.models import Source, Submission
+from securedrop_client.models import Source, Submission, Reply, User
```

Comment:
I foresee confusion between the sqlalchemy models and the SDK objects (not introduced in this PR, just a thought). Maybe we can change the names of the sqlalchemy objects to `SourceDatabaseObject` or something similar for the sake of being explicit (I don't love that name but you get the idea)